src/rabbit_mqtt_processor.erl

```diff
@@ -117,12 +119,15 @@ process_request(?CONNECT,
             _ ->
                 case creds(Username, Password, SSLLoginName) of
                     nocreds ->
+                        rabbit_core_metrics:auth_attempt_failed(Ip, <<"">>),
```

Comment:
I think that we should track the protocol in auth_attempt metrics. It would be `mqtt` in this case.